infra-as-code/bicep/modules/managementGroups/managementGroups.bicep

```diff
@@ -69,6 +72,11 @@ resource resTopLevelMg 'Microsoft.Management/managementGroups@2021-04-01' = {
   name: parTopLevelManagementGroupPrefix
   properties: {
     displayName: parTopLevelManagementGroupDisplayName
+    details: {
+      parent: {
+        id: parTopLevelManagementGroupParentId
```

Comment:
Can we add the logic back here that if an empty string ` ` is provided that we pass in `/providers/Microsoft.Management/managementGroups/${tenant().tenantId}`?

If any other string is provided im happy for the error, like our test just picked up 😄 
```
ERROR: {"status":"Failed","error":{"code":"DeploymentFailed","message":"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.","details":[{"code":"BadRequest","message":"{\r\n  \"error\": {\r\n    \"code\": \"BadRequest\",\r\n    \"message\": \"Invalid management group creation parameters: expected 'parentId' format: '/providers/Microsoft.Management/managementGroups/ID'\",\r\n    \"details\": null\r\n  }\r\n}"}]}}
```

If you can make the change, then we can get this one merged 👍